spec/content/automate/ManageIQ/Infrastructure/VM/Migrate/StateMachines/VMMigrate.class/__methods__/update_migration_status_spec.rb

```diff
@@ -0,0 +1,76 @@
+require_domain_file
+
+describe ManageIQ::Automate::Infrastructure::VM::Migrate::StateMachines::UpdateMigrationStatus do
+  let(:miq_server) { EvmSpecHelper.local_miq_server }
+  let(:user) { FactoryGirl.create(:user_with_email_and_group) }
+  let(:miq_request_task) do
+    FactoryGirl.create(:miq_request_task, :miq_request => request,
+                       :source => vm, :state => 'fred')
+  end
+  let(:request) { FactoryGirl.create(:vm_migrate_request, :requester => user) }
+  let(:ems) { FactoryGirl.create(:ems_vmware, :zone => FactoryGirl.create(:zone)) }
+  let(:vm) { FactoryGirl.create(:vm_vmware, :ems_id => ems.id, :evm_owner => user) }
+  let(:svc_model_vm) do
+    MiqAeMethodService::MiqAeServiceVm.find(vm.id)
+  end
+  let(:svc_model_miq_server)       { MiqAeMethodService::MiqAeServiceMiqServer.find(miq_server.id) }
+  let(:svc_model_miq_request_task) { MiqAeMethodService::MiqAeServiceMiqRequestTask.find(miq_request_task.id) }
+  let(:root_hash) do
+    { 'vm_migrate_task' => MiqAeMethodService::MiqAeServiceMiqRequestTask.find(miq_request_task.id) }
+  end
+
+  let(:root_object) do
+    obj = Spec::Support::MiqAeMockObject.new(root_hash)
+    obj["miq_server"] = svc_model_miq_server
+    obj
+  end
+
+  let(:ae_service) do
+    Spec::Support::MiqAeMockService.new(root_object).tap do |service|
+      current_object = Spec::Support::MiqAeMockObject.new
+      current_object.parent = root_object
+      service.object = current_object
+    end
+  end
+
+  shared_examples_for "#Update Migration Status" do
+    it "check" do
+      ae_service.root['ae_result'] = ae_result
+      allow(ae_service).to receive(:inputs) { {'status' => "fred"} }
```

Comment:
@billfitzgerald0120 I see that you're passing status into the shared example. Should status here be the passed in status?